build.sbt

```diff
@@ -206,7 +206,6 @@ lazy val publishSettings = Seq(
 )
 
 lazy val testSettings = Seq(
-  Test / parallelExecution := false,
```

Comment:
don't think we need serial test execution

Reply:
I can't recall why it was there. If they pass locally should be good.